Corona-Warn-App/src/main/res/values-de/dcc_reissuance_strings.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <!-- DCC Reissuance: Success Screen -->
+
+    <!-- XHED: DCC reissuance success screen title -->
+    <string name="dcc_reissuance_success_screen_title">Zertifikat aktualisiert</string>
+
+    <!-- YTXT: Headline for DCC reissuance success screen -->
+    <string name="dcc_reissuance_success_screen_headline">Ihr Zertifikat wurde erfolgreich aktualisiert</string>
+
+    <!-- YTXT: Body for DCC reissuance success screen -->
+    <string name="dcc_reissuance_success_screen_body">Sie finden das aktualisierte Zertifikat nun im Zertifikate Tab.</string>
```

Comment:
```suggestion
    <string name="dcc_reissuance_success_screen_body">Sie finden das aktualisierte Zertifikat nun in der Zertifikate-Registerkarte.</string>
```
According to Figma

Reply:
Hmm, in the iOS Figma Screen, it says "Zertifikate-Registerkarte", but on Android it is "Zertifikate Tab.". Which one should be used?

Reply:
For Android it's also Registerkarte, see here: https://www.figma.com/file/KuDEcMSREg3VhZRhC3FXTY/COVID19_Master_2.18?node-id=44407%3A33966
Is this screen somewhere else with "Tab"?
But it should definitely be Registerkarte, as we also already used it elsewhere to refer to this place.

Reply:
Aligned with Anna-Maria - This is the correct success screen: https://www.figma.com/file/KuDEcMSREg3VhZRhC3FXTY/COVID19_Master_2.18?node-id=44407%3A35035. The one you linked was outdated and not in the scope of the feature and is now hidden. Could you maybe align with her regarding the exact wording. Figma says "Tab".

Reply:
@KathrinLuecke any update?

Reply:
It should be "Registerkarte"- just agreed on it with Anna-Maria.